js/render.js

```diff
@@ -4477,11 +4477,10 @@ Renderer.creature = {
 			let renderStack = [];
 			for (let sc of cr.spellcasting) {
 				const meta = [];
-				let spellcastingName = sc.name ? sc.name : `${sc.tradition} ${sc.type}`.toTitleCase();
```

Comment:
Add ` Spells` here

Reply:
The issue was that it was appended to an already existing name, whether it was auto-generated or not.
As such, all that is needed is hard-code the `name` property to spells that have a different way of naming their spells, and only those. The rest that are just the spell tradition and type, can be auto generated with `${type} ${tradition} Spells`.

Reply:
Added as PR #303 for comparison.